app/domain/logs.rb

```diff
@@ -555,6 +555,41 @@ module AuthnJwt
         code: "CONJ00109D"
       )
 
+      FetchingIdentityPath = ::Util::TrackableLogMessageClass.new(
+        msg: "Fetching identity path...",
+        code: "CONJ00110D"
+      )
+
+      FetchedIdentityPath = ::Util::TrackableLogMessageClass.new(
+        msg: "Successfully fetched JWT identity path '{0-identity-path}'",
+        code: "CONJ00111I"
+      )
+
+      FetchingIdentityByInterface = ::Util::TrackableLogMessageClass.new(
+        msg: "Fetching JWT identity by interface: '{0-interface-name}'...",
+        code: "CONJ00112D"
+      )
+
+      FetchedIdentityByInterface = ::Util::TrackableLogMessageClass.new(
+        msg: "Successfully fetched identity '{0-identity}' by interface: '{1-interface-name}'",
+        code: "CONJ00113I"
+      )
+
+      AddingIdentityPrefixToIdentity = ::Util::TrackableLogMessageClass.new(
+        msg: "Adding identity prefix '{0-identity-prefix}' to identity '{1-identity}'...",
+        code: "CONJ00114D"
+      )
+
+      AddedIdentityPrefixToIdentity = ::Util::TrackableLogMessageClass.new(
+        msg: "Successfully added JWT identity prefix. Calculated identity is: '{0-identity}'",
```

Comment:
I'm curiuous about "Calculated". Is it really calculated? Maybe "Configured identity is"/"The full identity is:"

Reply:
We perform 2 manipulations on the identity:
1. Take the identity string from the token
2. add the prefix from configuration
3. add "host" in order to block human users 

So if in the future we will have more steps the term "full" can be confusing